src/components/calcite-slider/calcite-slider.scss

```diff
@@ -159,6 +159,9 @@ $tick-height: 4px;
   top: 0;
   height: $track-height;
   background-color: var(--calcite-ui-blue-1);
+  &:hover {
+    cursor: ew-resize;
```

Comment:
I think I worded my feedback before poorly - it's weird (IMO YMMV) to have this cursor for sliders that aren't a range, since you aren't actually sliding anything, you're just placing the single handle at the end of your click / touch. Maybe just show when the slider has two handles and you are actually dragging the range?

Reply:
Ah, good catch!  I didn't realize this was applying to the bar all the time.  I'll fix that right up.